include/comingsoon.css

```diff
@@ -0,0 +1,10 @@
+.bgimg {
+background-image: url('http://www.dreamalittlebigger.com/wp-content/uploads/2013/08/gold-hi-res-glitter-freebie-dream-a-little-bigger.jpg?x32341');
```

Comment:
This works from a technical standpoint, but it's always a bit risky including images hosted on someone else's website because they could change the image at any time. Instead, try saving that image to your own site and using that.

It's worth noting that in theory this image might be copyrighted in which case you aren't necessarily allowed to use it. Since this is just a practice site that's probably not worth worrying about, but if you ever make a website for commercial purposes, we should talk about this.